GLib-2.0.gir

```diff
@@ -54015,6 +54015,11 @@ called with the same type of fixture (the @data_size argument), but varying
         </parameter>
       </parameters>
     </function>
+    <function name="test_disable_crash_reporting" c:identifier="g_test_disable_crash_reporting">
```

Comment:
They have a since annotation upstream, the usual g-i update dance is needed. Sad me.